README.md

```diff
@@ -113,6 +113,196 @@ The below Environment variables are optional. If not provided a manual Applicati
 |`BOLD_SERVICES_USER_EMAIL`|Yes|It should be a valid email.|
 |`BOLD_SERVICES_USER_PASSWORD`|Yes|It should meet our password requirements.|
 
+## Environment variables for configuring `Branding` in backend
+The below Environment variables are optional. If not provided Bold BI will take the default configured values.
+
+<table>
+   <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_MAIN_LOGO
+      </td>
+      <td>   
+       This is header logo for the applicationand the preferred image size is 40 x 40 pixels.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_LOGIN_LOGO
+      </td>
+      <td>     
+       This is login logo for the application and the preferred image size is 200 x 40 pixels.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_EMAIL_LOGO
+      </td>
+      <td>     
+       This is email logo, and the preferred image size is 200 x 40 pixels.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_FAVICON
+      </td>
+      <td>     
+       This is favicon and the preferred image size is 40 x 40 pixels. 
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_FOOTER_LOGO
+      </td>
+      <td>     
+       This is powered by logo and the preferred size is 100 x 25 pixels.
+       <br />
+       <br />
+       <b>Note:</b><br/>• All the branding variables are accepted as URL.<br/>• <b>Ex:</b> https://example.com/loginlogo.jpg <br/>• <b>Image type:</b> png, svg, jpg, jpeg.<br/>• If you want to use the custom branding, provide the value for all branding variables If all variable values are given, application will use the branding images, otherwise it will take the default logos. 
```

Comment:
All branding variables are accepted as URL.<br/>• <b>Ex:</b> https://example.com/loginlogo.jpg <br/>• <b>Image type:</b> png, svg, jpg, jpeg.<br/>• If you want to use custom branding, provide the value for all branding variables. If all variable values are given, the application will use the branding images, otherwise, it will take the default logos.

Reply:
concern addressed